### DISCUSSION TITLE
(DNS zone(s) are not found). I just wanted to confirm that I can't use this with Cloudflare DNS

### DISCUSSION THREAD

**Discussion by v1xiii** [shibayan/appservice-acmebot#453] [Q&A] [answered]
I apologize for not knowing what I'm doing with this stuff, but I have everything installed on Azure and when I try to create a cert I get the following error:

Orchestrator function 'AddCertificate_Orchestrator' failed: The orchestrator function 'IssueCertificate' failed: "The activity function 'Dns01Precondition' failed: "DNS zone(s) are not found. DnsNames = tcgadvance.com". See the function execution logs for additional details.". See the function execution logs for additional details.

From what I've read, it sounds like I simply can't use appservice-acmebot since my DNS is not handled by Azure. Can someone confirm? 

Seems like I'm going to have to run certificates manually for the time being since the Azure app managed certificates mysteriously fail when I try to use those as well.

**Comment by shibayan** [comment, marked as answer]
The App Service version of Acmebot can only use Azure DNS and does not support Cloudflare. Please use the Key Vault version.
